### DISCUSSION TITLE
Respositories on github page when i login

### DISCUSSION THREAD

**Discussion by garyanson** [community/community#23767] [New to GitHub]
<p>When I login to github I see the following repositories in the left column</p>
<p><img src="https://user-images.githubusercontent.com/27372147/181079263-aacd49d1-dc2a-4479-a001-b6ac393c9f90.png" alt="2020-06-11 13.54.47" data-base62-sha1="5YBQQ1ld36GCx3mxY32OYjPwr3c" width="322" height="213"></p>
<p>How do I get rid of some of them that aren’t mine?</p>
<p>Thanks very much!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/garyanson">@garyanson</a>, thank you for being here and welcome to the community! The list of repositories shown on the left side of your dashboard shows repositories that you own or have contributed to. We don’t offer a way to clear or customize this listing, but I will submit this as a feature request to the team.</p>
